old2new_gcl.GCL.R

```diff
@@ -0,0 +1,61 @@
+old2new_gcl.GCL <- function (sillyvec, overwrite = FALSE){
+  #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  # This function converts the old style "*.gcl" objects from nested arrays to a tibbles.
+  # The old style objects can be saved as *.gcl_old before they are overwritten.
+  #
+  # Inputs~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  #   sillyvec - A character vector of silly codes with associated *.gcl objects that need to be converted.
+  #   overwrite - logical; whether you want the old objects overwritten without saving 
+  #              (TRUE) or assign the new objects to *.gcl_old before overwriting (FALSE) 
```

Comment:
Always overwrites old .gcl object regardless of "overwrite" logical value. No *.gcl_old object created with "FALSE" or empty.